integration/test_batch.py

```diff
@@ -132,6 +132,28 @@ def test_delete_objects(client: weaviate.Client):
     assert "four" in names
     assert "five" in names
 
+    with client.batch as batch:
```

Comment:
can we have a test that parametrizes the filters for batch delete and makes sure that all combinations work? Just like with normal filtering?